docs/GettingStarted.md

```diff
@@ -946,6 +947,29 @@ alternate_client = Demo::Echo::Service.rpc_stub_class.new(
 
 The integration will ensure that the `configured_interceptor` establishes a unique tracing setup for that client instance.
 
+### Kafka
+
+The Kafka integration provides tracing of the `ruby-kafka` gem:
+
+You can enable it through `Datadog.configure`:
+
+```ruby
+require 'ddtrace'
```

Comment:
I think we should document here that the user needs to have enabled `ruby-kafka` instrumentation enabled for `ddtrace` to work. A suggested way to do this (but please feel free to adapt as you see fit):

```suggestion
require "active_support/notifications" # Enables 'ruby-kafka' instrumention
require "kafka"
require 'ddtrace'
```